Compiling builds after merge of #2748 now showing warning messages

I'm not sure if this is a problem, but since the merge of #2748 Add protection against unconnected VCP , a warning message appears part way through the build process. This is showing up in all builds I have made since that pull request was merged.

Building SPRACINGF3EVO

./src/main/drivers/serial.c: In function 'serialIsConnected':
./src/main/drivers/serial.c:109:39: warning: passing argument 1 of 'instance->vTable->isConnected' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
         instance->vTable->isConnected(instance);
                                       ^~~~~~~~
./src/main/drivers/serial.c:109:39: note: expected 'serialPort_t * {aka struct serialPort_s *}' but argument is of type 'const serialPort_t * {aka const struct serialPort_s *}'

This is also showing up in all The Travis CI builds listed since that date.
The build completes successfully and seems to work properly. 
If this is not an issue  please comment and close this issue

--- a/src/main/drivers/serial.h
+++ b/src/main/drivers/serial.h
@@ -86,7 +86,7 @@ struct serialPortVTable {
 
     void (*writeBuf)(serialPort_t *instance, const void *data, int count);
 
-    bool (*isConnected)(serialPort_t *instance);
+    bool (*isConnected)(const serialPort_t *instance);
 
     // Optional functions used to buffer large writes.
     void (*beginWrite)(serialPort_t *instance);

--- a/src/main/drivers/serial_usb_vcp.c
+++ b/src/main/drivers/serial_usb_vcp.c
@@ -92,7 +92,7 @@ static uint8_t usbVcpRead(serialPort_t *instance)
     }
 }
 
-static bool usbVcpIsConnected(serialPort_t *instance)
+static bool usbVcpIsConnected(const serialPort_t *instance)
 {
     (void)instance;
     return usbIsConnected() && usbIsConfigured();

